homeassistant/components/knx/storage/knx_selector.py

```diff
@@ -0,0 +1,80 @@
+"""Selectors for KNX."""
+
+from enum import Enum
+from typing import Any
+
+import voluptuous as vol
+
+from ..validation import ga_validator, maybe_ga_validator
+
+
+class GASelector:
```

Comment:
What's GA?

Reply:
It is a common (in the KNX world) abbreviation for "Group Address". That's used to assign KNX telegram payloads to specific entity functions - and thus the main thing we need a user to configure (1-2 for each function - set and state address). Thats vaguely comparable to an MQTT topic or Modbus register address.